### PR TITLE
add_shared_pref for dark mode

### PR DESCRIPTION
**Pull request by mysticeti**

### CODE REVIEW

File: lib/app_settings.dart [hunk 1]
```diff
@@ -1,15 +1,30 @@
...
   }
 
   get getCurrentThemeMode {
-    return currentThemeMode;
+    if (isFirstRun) {
+      isFirstRun = false;
+      return currentThemeMode;
+    } else {
+      return currentThemeMode;
+    }
```
Comment (joykuotw) [inline lib/app_settings.dart:28]: Why need `isFirstRun`? there is no difference.

File: lib/main.dart [hunk 2]
```diff
@@ -132,6 +144,7 @@ class _MyAppState extends State<MyApp> {
   final FirebaseAnalytics analytics = FirebaseAnalytics();
   final _pageController = PageController(initialPage: 0);
   final fcmDemoPage = 12;
+  bool isFirstRun = true;
```
Comment (joykuotw) [inline lib/main.dart:147]: not used.

**Review by joykuotw — CHANGES_REQUESTED** [review]

File: lib/app_settings.dart (hunk 1 above)
Comment (mysticeti) [reply lib/app_settings.dart:28]: Removed it. It does not do anything

File: lib/main.dart (hunk 2 above)
Comment (mysticeti) [reply lib/main.dart:147]: Removed in the next commit. Thank you for pointing it out.

**Review by joykuotw — APPROVED** [review]

**Pull request merged**